woocommerce/woocommerce-android
# Remove order creation m1 feature flag

Summary
==========
Fixes issue #5694 by removing the Feature flag for the Order creation M1 features, making available creating an Order with the Address, Product selection, and Payments cost summary using local price calculation.


How to Test
==========
Just do overall feature tests including verifications if the feature works as expected excluding the newly merged work around the API price calculation and still follows the Experimental toggle state to make it available.


Update release notes:
